core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -880,8 +880,8 @@ class TaskSetManagerSuite extends SparkFunSuite with LocalSparkContext with Logg
     assert(manager.resourceOffer("execB", "host2", ANY).get.index === 3)
   }
 
-  test("speculative task should not run on a given host where another attempt " +
-    "is already running on") {
+  test("SPARK-23888: speculative task should not run on a given host " +
+    "where another attempt is already running on") {
```

Comment:
I'd reword this to be a bit more specific to what you're trying to test: 

speculative task cannot run on host with another running attempt, but can run on a host with a failed attempt.

Reply:
Sure. Also, do we need to reword PR and jira title? @squito